C-Plus-Plus/math/Extended_Euclidean_Algorithm.cpp

```diff
@@ -17,12 +18,40 @@ int gcd (int a, int b, int &x, int &y)
 	return d;
 }
 
+// Iterative version of Extended Euclidean algorithm
+
+int gcd_iterative(int a, int b, int& x, int& y) {
+    x = 1, y = 0;
+    int x1 = 0, y1 = 1, a1 = a, b1 = b;
+    while (b1) {
+        int q = a1 / b1;
+        tie(x, x1) = make_tuple(x1, x - q * x1);
+        tie(y, y1) = make_tuple(y1, y - q * y1);
+        tie(a1, b1) = make_tuple(b1, a1 - q * b1);
+    }
+    return a1;
+}
+
 int main()
 {
     int x = 0, y = 0, a, b;
     cout<<"Enter the numbers whose GCD is to be found:"<<endl;
     cin>>a>>b;
     cout<<"GCD of "<<a<<" and "<<b<<" is:"<<endl;
-    cout<<gcd(a, b, x, y);
```

Comment:
Do not remove this line . Just print the result with both the methods mentioning the method name.
Like: ```cout<<"GCD (iterative method) is : "<<gcd_iterative(a, b, x, y)```

Reply:
@atarax665 Please review, done with the requested changes